api/src/main/java/org/openmrs/api/ProviderService.java

```diff
@@ -331,4 +335,77 @@ public List<Provider> getProviders(String query, Integer start, Integer length,
 	 */
 	@Authorized( { PrivilegeConstants.GET_PROVIDERS })
 	public Provider getUnknownProvider();
+
+	/**
+	 * Gets all Provider Roles in the database
+	 *
+	 * @param includeRetired whether to include retired provider roles or not
+	 * @return list of all provider roles in the system
+	 * 
+	 * @since 2.8.0
+	 */
+	@Authorized(value = {PROVIDER_ROLE_API_PRIVILEGE, PROVIDER_ROLE_API_READ_ONLY_PRIVILEGE}, requireAll = false)
+	public List<ProviderRole> getAllProviderRoles(boolean includeRetired);
```

Comment:
This seems to deviate from our conventions. Can we use the same privilege pattern as for the rest of the services?